src/Processor/Shortcode/assets/code-groups.css

```
@@ -1,30 +1,53 @@
 .code-group {
+    --code-group-accent: var(--c-link, var(--link-color, #2563eb));
+    --code-group-border: var(--c-border-strong, var(--border-color, #d1d5db));
+    --code-group-tab-hover: rgb(127 127 127 / 10%);
     margin: 1.5rem 0;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)